assets/src/stories-editor/helpers/index.js

```diff
@@ -758,56 +758,21 @@ export const addBackgroundColorToOverlay = ( overlayStyle, backgroundColors ) =>
 };
 
 /**
- * Object of block attributes to set to default when inserting a template.
- */
-const emptyTemplateMapping = {
-	// @todo This can use just arrays of attribute keys instead of object.
-	'amp/amp-story-text': {
-		content: '',
-	},
-	'amp/amp-story-page': {
-		mediaUrl: null,
-		mediaType: null,
-		focalPoint: {},
-	},
-	'core/image': {
-		url: null,
-		positionLeft: null,
-	},
-	'amp/amp-story-cta': {
-		text: null,
-		link: null,
-	},
-	'core/quote': {
-		citation: null,
-		value: null,
-	},
-};
-
-/**
- * Gets a skeleton template block from pre-populated block.
+ * Resets a block's attributes except for a few ones relevant for the layout.
  *
- * @param {Object}   block            Original block.
- * @param {Object}   block.name       Block name.
- * @param {Object[]} block.attributes Block attributes.
+ * @param {Object} block Block object.
+ * @param {Object} block.attributes Block attributes.
  *
- * @return {Object} Block object.
+ * @return {Object} Filtered block attributes.
  */
-const getSkeletonTemplateBlock = ( block ) => {
-	if ( ! emptyTemplateMapping[ block.name ] ) {
-		return block.attributes;
-	}
-
+const resetBlockAttributes = ( block ) => {
 	const attributes = {};
-	each( block.attributes, function( value, key ) {
-		if ( undefined === emptyTemplateMapping[ block.name ][ key ] ) {
-			attributes[ key ] = value;
-		}
-	} );
+	const attributesToKeep = [ 'positionTop', 'positionLeft', 'width', 'height', 'ampFitText', 'tagName' ];
```

Comment:
Wondering if it makes sense to keep `ampFitText` unless we're keeping the font size as well. Otherwise, if the area is very small and `ampFitText` is set to `false` but there is no font size assigned then it would be the default font size which would be too large. Maybe we could either remove `ampFitText` as well (basically setting it to default which is `true`) or alternatively keep the font size, too. Thoughts?

Reply:
I am a bit on the fence with regards to where to draw the line with all these attributes. For example, font family also affects these things.

Given that `ampFitText` gives for a much better editing experience, I am inclined to remove that one here too though.